src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
This change *undoes* #14, which is a fix we want.  My impression was that this was supposed to be included in SOFA, but apparently not... but either way, we do *not* want to make this particular change.  So can you just put back the 
``da = changes[(i < 0) ? 0 : i].delat;`` that was there before?